src/main/resources/ceasar.sql

```diff
@@ -0,0 +1,214 @@
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
+
+DROP SCHEMA IF EXISTS `сeasar` ;
+
+CREATE SCHEMA IF NOT EXISTS `сeasar` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci ;
+USE `сeasar` ;
```

Comment:
Found solution for my problem seems like you have russian character 'c' in schema name or something like that